src/Apib.php

```diff
@@ -78,6 +78,7 @@ private function processApiHttpTransactions(
         $req = new SpasRequest();
 
         if ($apiResourceGroup->getTitle()) {
+            $req->setResourceGroup($apiResourceGroup->getTitle());
```

Comment:
According to my suggestion in https://github.com/hendrikmaus/spas-parser/pull/2/files#diff-85bbfd9c8354c84ff7bb5d1b61c8ffedR28 we should pull `$req->setResourceGroup($apiResourceGroup->getTitle());` out of the if condition here, change it to `$req->setResourceGroup($apiResourceGroup);` and call it above the `if` condition.

This way, every request would carry a reference to the group it was in.